packages/ounit2/ounit2.2.2.6/opam

```diff
@@ -14,6 +14,9 @@ depends: [
   "seq"
   "stdlib-shims"
 ]
+conflicts: [
+  "ounit" {!= version}
```

Comment:
This would not work. In your second example `ounit 2.1.2` is installed allongside ounit2 2.2.6.
So the `!=version` would not prevent this to happen

Reply:
Ah no, sorry, it is in the conflicts, so in principle would be fine